src/js/view/controls/controlbar.js

```diff
@@ -270,7 +275,12 @@ export default class Controlbar {
 
         // Listen for model changes
         _model.change('volume', this.onVolume, this);
-        _model.change('mute', this.onMute, this);
+        _model.change('mute', (model, muted) => {
+            this.renderVolume(muted, model.get('volume'));
+            const muteText = model.get('mute') ? localization.unmute : localization.mute;
```

Comment:
Instead of reading `model.get('mute')` again, you can re-use the `muted` parameter.